tools/checkcomments/checkcomments.go

```diff
@@ -59,3 +70,71 @@ func run(pass *analysis.Pass) (any, error) {
 
 	return nil, nil
 }
+
+// isIssueopen check the issue open or closed.
+func isIssueOpen(todoText string) bool {
+	issueURL := getURL(todoText)
+	if issueURL == "" {
+		return false
+	}
+
+	token := ""
+	ctx := context.Background()
+	httpClient := oauth2.NewClient(ctx, oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	))
+
+	client := github.NewClient(httpClient)
+
+	owner := "owner"
+	repo := "repos"
```

Comment:
```suggestion
	owner := "FerretDB"
	repo := "FerretDB"
```

`owner` and `repo` in the documentation are placeholders that should be replaced by repository owner (user or organization name) and repository name.